authentik/lib/utils/convert.py

```diff
@@ -0,0 +1,34 @@
+"""Conversion utilities for data type normalization."""
+
+from typing import Any
+
+
+def ensure_string_id(id_value: Any) -> str | None:
```

Comment:
Isn't this return type misleading? Non integers pass through unchanged, so could should be:

```suggestion
def ensure_string_id(id_value: Any) -> Any:
```